java/src/jmri/jmrix/can/cbus/CbusConstants.java

```diff
@@ -17,6 +20,24 @@ public final class CbusConstants {
     public static final int DEFAULT_STANDARD_ID = 0x7a;
     public static final int DEFAULT_EXTENDED_ID = 0x7a;
 
+    /**
+     * Get a String Array of valid CBUS CAN IDs for use with fixed IDs.
+     * CBUS dev guide 6b was 120-127, 6c is 100-127.
+     * @return list of CAN IDs from 100 to 127
+     */
+    public static String[] getValidFixedCanIds() {
+        String[] result = new String[28];
+        for (int i = 0; i < 28; i++) {
```

Comment:
No need to change, but a suggestion.

If you add a line like:

    final int MAX = 28;

Then you can use NUM instead of 28 in the code. The reason I suggest this is that the two numbers depend on each other. If someone by mistake changes line 30 but not line 29, he would get an exception.